### ISSUE TITLE
more concrete example?

### ISSUE THREAD

**Issue by ClioCJS** [Cormanz/smartgpt#10]
Would it be possible to provide a more concrete example of how to use it, in the README file?

Like an example where we ask it a specific question and save the result as a file (though STDOUT would be good enough, honestly!).

Alas, the README was just slightly over my head, but only slightly.

**Comment by Cormanz** [comment]
I think that's a great idea. I'm planning to significantly overhaul the README, so I'll try to simplify it and provide more useful use-cases. I do want to note though that this is still a very experimental project, so a lot of our time is mostly spent on rapid prototyping rather than polishing and showing uses.

Thanks for your enthusiasm!

**Comment by orvitpng** [comment]
> I think that's a great idea. I'm planning to significantly overhaul the README, so I'll try to simplify it and provide more useful use-cases. I do want to note though that this is still a very experimental project, so a lot of our time is mostly spent on rapid prototyping rather than polishing and showing uses.
> 
> Thanks for your enthusiasm!

I recommend documenting outside the README. README should be for general knowledge that everyone should know for this.

**Comment by Cormanz** [comment]
> I recommend documenting outside the README. README should be for general knowledge that everyone should know for this.

I agree. In fact, a comment in #7 mentioning progress before finishing it mentions the same goal.
